test.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  tests:
+    build:
+      context: .
+      target: test
+    image: ov-tests
+    volumes:
+      - ./:/app/
+    entrypoint: /app/docker_entrypoints/test.sh
+    environment:
+      - OV_DB_ENGINE=django.db.backends.sqlite3
+      - OV_DB_NAME=testdb
```

Comment:
I can see where the top-level `dev.yml` is used in the `ov` script, but i'm not seeing where `dev.yml` is used.

Reply:
Forthcoming: https://github.com/WGBH-MLA/ov-wag/pull/83/files#diff-fe4c6e543bb224b046d34d6c9c3f9600162e576dc2512c670e91998dff2511a1

Though, it is incorrectly referred to as `tests.yml`. Will fix it there.